controller/app/task/task-imu.cpp

```diff
@@ -73,6 +73,11 @@ static constexpr LSM6DS::Config s_imu_config =
 static LSM6DS s_imu(s_imu_config);
 static ComplementaryFilter s_complementary_filter;
 
+
+static float* s_accel_out;
```

Comment:
this is really wrong :/. We shouldn't use pointers like this. Add the get function into a struct and return a copy value (new or reference here). This is way safer and we can add later a mutex to have some kind of "atomic" operation.